docs/protocols.md

```diff
@@ -92,13 +92,17 @@ The CQL binary protocol is Cassandra's and Scylla's main client-facing
 protocol. Scylla supports several Scylla-only extensions to this protocol,
 described in [protocol-extensions.md](protocol-extensions.md).
 
-By default scylla listens to the CQL protocol on port 9042, which can be
-configured via the `native_transport_port` configuration option, or entirely
-disabled by setting the `start_native_transport` option to 0.
-Again these option names were chosen for backward-compatibility with Cassandra
-configuration files: They refers to CQL as the "native transport", to contrast
-with the older Thrift protocol (described below) which wasn't native to
-Cassandra.
+By default, Scylla listens to the CQL protocol on port 9042, which can be
+configured via the `native_transport_port` configuration option. Scylla also
+supports the CQL protocol via TLS/SSL encryption, which can be enabled via the
+`native_transport_port_ssl` configuration option (default port is 9142). Users
```

Comment:
This sentence is ambiguous. The phrase "can be enabled via..." suggests it's not enabled by default and becomes enable if you set this `native_transport_port_ssl` option - but then you go and say it is actually set to 9142 by default. So does SSL listen to port 9142 or not? Or maybe the real story is different - and other options (setting the certificate, etc.) are what actually enables ssl? I think we need a clearer (and possibly more correct) explanation here on what is really needed to enable SSL.